src/Analysis/Engine/Impl/Values/Definitions/IFunctionInfo.cs

```diff
@@ -23,6 +23,7 @@ public interface IFunctionInfo: IAnalysisValue {
         bool IsClassMethod { get; }
         bool IsProperty { get; }
         bool IsClosure { get; }
+        bool IsAbstract { get; }
```

Comment:
Since these are public interfaces you may have to create `IFunctionInfo2` deriving from `IFunctionInfo` and add property there. This way code that uses public LS extensibility API will continue working. Yes, there is at least one VS Code extension that uses the API.